pkg/networkmanager/interfaces.js

```diff
@@ -957,7 +959,8 @@ export function NetworkManagerModel() {
             deactivate: function() {
                 return call_object_method(get_object("/org/freedesktop/NetworkManager", type_Manager),
                                           "org.freedesktop.NetworkManager", "DeactivateConnection",
-                                          objpath(this));
+                                          objpath(this))
+                    .then(results => undefined);
```

Comment:
This is to signify that we don't expect any return value, and don't parse the empty list?

Reply:
Exactly.